portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -12,7 +14,29 @@ public class HelloWorldServlet extends HttpServlet {
   //GET request handler
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Cindy!</h1>");
+    ArrayList<String> comments = new ArrayList<String>();
+    comments.add("I'm learning a lot of new concepts!");
+    comments.add("Everybody is very supportive!");
+    comments.add("Relearning Java right now");
+
+    String json = convertToJSON(comments);
+
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+
+
   }
+    private String convertToJSON(ArrayList<String> commentsArr){
+        String json = "{";
+        json += "\"comments\": ";
+        json += "[";
+        for(int i = 0; i < commentsArr.size()-1; i++){
+            String str = commentsArr.get(i);
+            json += "\"" + str + "\"" + ", ";
+        }
+        json += "\"" + commentsArr.get(commentsArr.size()-1) + "\"";
+        json += "]";
+        json += "}";
+        return json;
+    }
```

Comment:
You might want to use a JSON library like gson https://www.java2novice.com/java-json/google-gson/map-to-json/#:~:text=Nginx%20Basics-,How%20to%20convert%20map%20to%20json%20string%20using%20Gson%20APIs%3F,-This%20page%20shows